docs/2.10.0/docs/canton/usermanual/packagemanagement.rst

```diff
@@ -61,235 +61,181 @@ Understanding Package Vetting
 -----------------------------
 
 Every participant operator uploads DARs individually to their participant node. There is no global DAR repository
-anywhere and participants do not have access to each others DAR repositories. Therefore, for two participants to
-synchronise on a transaction that uses packages contained in a certain DAR, we need both participant operators to
-have uploaded the same DAR before the transaction was submitted.
+anywhere and participants do not have access to each others DAR repositories. Instead, participants publicly declare towards their peers
+that they are ready to receive transactions referring to certain packages.
+This is done by Canton package state topology transactions which can be seen by all the participants connected to a specific sync domain.
+Therefore, for two participants to synchronize on a transaction that uses packages contained in a certain DAR, both participant operators
```

Comment:
Mentioning the DAR here is slightly imprecise because the checks that are done by Canton are done on the package level, not on the DAR level.

Reply:
Alright, let's keep as is.